### PR TITLE
Jetty 11 module

### PR DESCRIPTION
**Pull request by shakuzen**
I've copied just `TimedHandler` and adapted to necessary changes for Jetty 11. I originally copied all of the classes in the same package, but I'm not sure there is any problem using the ones compiled against Jetty 9 with Jetty 11, since none of them contain `javax` package imports and the relevant Jetty API seems to be unbroken.

I have added some Jakarta-specific classes to Micrometer core for tagging, mirroring the corresponding javax servlet classes.

Jetty 11 is Java 11 baseline, so this module is also Java 11 baseline, which isn't a problem for our build anymore since we dropped the Java 8 CI build.

See #3234

### CODE REVIEW

File: micrometer-jetty11/src/test/java/io/micrometer/jetty11/TimedHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,435 @@
+/*
+ * Copyright 2019 VMware, Inc.
```
Comment (jonatan-ivanov) [inline micrometer-jetty11/src/test/java/io/micrometer/jetty11/TimedHandlerTest.java:2]: Should we keep the copyright year in this case?

**Comment by shakuzen** [comment]
@tipsy if you have time to review this, we'd appreciate any feedback.

**Comment by shakuzen** [comment]
We can still see what the Jetty team thinks about packaging and releasing this code in a Jetty module, but I wanted to put together what it would be here first. It's mostly copy-paste from the Jetty 9 support with import changes.

**Comment by tipsy** [comment]
> @tipsy if you have time to review this, we'd appreciate any feedback.

@zugazagoitia checked it out on my behalf and tested it against Javalin 5.0.0.RC1 - everything appears to be working fine :)

**Comment by shakuzen** [comment]
I copied this over with the Timer-based instrumentation, but proybably it should be done with the ObservationOrTimerInstrumentation instead. We would need to add a builder to optionally pass the ObservationRegistry, and then the question is whether to leave public constructors (from the copied TimedHandler) or not.

**Comment by shakuzen** [comment]
> probably it should be done with the ObservationOrTimerInstrumentation instead.

I'll have to look at that in a follow-up issue. I don't want to block getting this into the upcoming release on that.

**Pull request merged**